src/components/LeftSidebar/Sponsors.astro

```diff
@@ -24,6 +24,19 @@
 				fill="currentColor"></path>
 		</svg>
 	</a>
+	<a
+		href="https://idx.dev/?utm_source=astro&utm_medium=astro&utm_campaign=astro"
+		aria-label="Google IDX"
+	>
+	<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 214 52">
```

Comment:
Needs a `width` attribute to size properly and the `aria-hiddent="true"` hides the SVG from screen readers (which get their label from the `aria-label` on the link element instead.

```suggestion
	<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 214 52" width="125" aria-hidden="true">
```